frontend/src/pages/profile/libs/enums/activity-thresholds.enum.ts

```diff
@@ -0,0 +1,7 @@
+const ActivityThresholds = {
+  threshold1: 0,
+  threshold2: 1,
+  threshold3: 3,
+} as const;
```

Comment:
Also rename filename
```suggestion
const ActivityThreshold = {
  FIRST: 0,
  SECOND: 1,
  THIRD: 3,
} as const;
```

Reply:
Was not resolved